### DISCUSSION TITLE
从antdv4升级到antdv5出现 not found /antd/dist/antd.less

### DISCUSSION THREAD

**Discussion by xiaogonggong-w** [ant-design/ant-design#40804] [Q&A]
<img width="858" alt="image" src="https://user-images.githubusercontent.com/79799040/219940754-037c0e64-b477-4ebb-9592-d09bbbeb0955.png">

现在antd5不是采用的是cssinjs么,所以在antd里面应该没有less文件,但是我项目里面也没有引入这个文件,目前只在.umi/umi.ts中出现这个.
<img width="639" alt="image" src="https://user-images.githubusercontent.com/79799040/219940823-f22f1397-d3e8-4444-9961-55f6a5d4808b.png">

所以,广大的朋友们,有谁遇到过这个问题么,应该怎么解决

**Comment by li-jia-nan** [comment]
https://ant.design/docs/react/migration-v5-cn#%E6%8A%80%E6%9C%AF%E8%B0%83%E6%95%B4

**Reply by xiaogonggong-w** [reply]
> https://ant.design/docs/react/migration-v5-cn#%E6%8A%80%E6%9C%AF%E8%B0%83%E6%95%B4

我就是按照这个来的,我没有引入任何antd的less,只是引入
<img width="605" alt="image" src="https://user-images.githubusercontent.com/79799040/219941691-dc41149b-9f18-4d73-bb74-4bdb8b7f6c05.png">
antd/dist/reset.css

**Reply by li-jia-nan** [reply]
不是很熟悉 umi，要么去 umi 仓库下面提个 issue？

**Reply by xiaogonggong-w** [reply]
好的,🙏

**Comment by yoyo837** [comment]
升级umi

**Reply by xiaogonggong-w** [reply]
现在umi版本是3.5.0,还需要升级
